src/components/utils/src/threads/thread_delegate_win.cc

```diff
@@ -29,7 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#if defined(WIN_NATIVE)
+#if defined(OS_WINDOWS)
```

Comment:
This wills will be out of  buidl in non-windows build? do we really need to add this define?
IF else, add build error with information about wrong including.